shell/platform/android/io/flutter/plugin/platform/VirtualDisplayController.java

```diff
@@ -199,6 +210,18 @@ public void dispose() {
     renderTarget.release();
   }
 
+  @TargetApi(21)
+  private void resizeAboveAndroidS(
+      View embeddedView, int width, int height, final Runnable onNewSizeFrameAvailable) {
+    renderTarget.resize(width, height);
+    // https://android.googlesource.com/platform/prebuilts/fullsdk/sources/android-30/+/refs/heads/master/android/app/Presentation.java#293
```

Comment:
Rewrite this comment block as:

// On Android versions 31+ resizing of a Virtual Display's Presentation is natively supported.

Reply:
Please address this comment.